src/components/ComponentsContext/components/Link/Link.js

```diff
@@ -14,13 +14,29 @@
  */
 
 import React from 'react';
+import isPropValid from '@emotion/is-prop-valid';
 
 import { childrenPropType } from '../../../../util/shared-prop-types';
 
 /**
  * A barebones Link component that's basically just an `<a>` tag
  */
-const Link = ({ children, ...props }) => <a {...props}>{children}</a>;
+const Link = ({ children, ...props }) => (
+  <a {...filterValidProps(props)}>{children}</a>
+);
+
+function filterValidProps(props) {
+  const INVALID_HTML_ATTRIBUTES = ['size', 'disabled', 'selected'];
```

Comment:
These props are specific to the Button component, but the Link component is used in other places than the Button component. Please move this logic to the button component [here](https://github.com/sumup-oss/circuit-ui/blob/canary/src/components/Button/components/RegularButton/RegularButton.js#L218) and [here](https://github.com/sumup-oss/circuit-ui/blob/canary/src/components/Button/components/PlainButton/PlainButton.js#L52).

Reply:
I've removed this blacklist because it was a "nice to have" feature and wasn't contributing to the PR goal.